module/shared-kernel/tests/Domain/ValueObject/CodeTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\SharedKernel\Tests\Domain\ValueObject;
+
+use Ergonode\SharedKernel\Domain\ValueObject\Code;
+use PHPUnit\Framework\TestCase;
+
+class CodeTest extends TestCase
```

Comment:
I'd suggest testing the abstraction(i.e. via anonymous class), not dummy implementation